tests/test_placeholder_validator.py

```diff
@@ -189,3 +190,62 @@ def test_validation_answer_and_transform_unit_match(
         arguments=arguments, transform_type=transform_type
     )
     assert validator.errors == expected_error
+
+
+def test_validation_answer_and_transform_unit_two_source_answers_mismatch():
+    filename = "schemas/invalid/test_invalid_calculated_summary_units_mismatch.json"
+    schema_file = _open_and_load_schema_file(filename)
+    validator = PlaceholderValidator(schema_file)
+    validator.validate()
+    assert validator.errors == [
+        {
+            "message": error_messages.ANSWER_UNIT_AND_TRANSFORM_UNIT_MISMATCH.format(
+                answer_unit="length-centimeter", transform_unit="length-meter"
+            ),
+            "identifier": "unit-total-playback",
```

Comment:
as the two error messages are identical, would another parameter for the mismatching answer id be a good idea here? so we'd have identifier and answer_id